src/nuts.py

```diff
@@ -22,16 +22,14 @@ def nuts():
 
 @nuts.command()
 @click.argument("path_to_shapes")
-@click.argument("path_to_attributes")
 @click.argument("path_to_output")
-def merge(path_to_shapes, path_to_attributes, path_to_output):
```

Comment:
I understand you are using a shapefile which contains attributes and polygons? Are you sure this works for all within [2006, 2010, 2013, 2016]?

Reply:
I'm not, no, but will check. Would you want this as a test?

Reply:
At least a manual check, yes. I _think_ this wasn't the case for NUTS2013. But maybe I just didn't see it.